src/api/dedicated/server/biosSettings/dedicated-server-biosSettings.v6.service.js

```diff
@@ -0,0 +1,39 @@
+angular.module('ovh-api-services').service('OvhApiDedicatedServerBiosSettingsV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiDedicatedServerBiosSettingsV6');
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+
+      return response.resource;
+    },
+  };
+
+  const dedicatedServerResource = $resource('/dedicated/server/:serverName/biosSettings', {
+    serverName: '@serverName',
+  }, {
+    get: { method: 'GET', cache },
+    getFeature: {
+      url: '/dedicated/server/:serverName/biosSettings/:featureName',
```

Comment:
Can't find `featureName` placeholder into the API Endpoint.

Is it still a draft?

Reply:
`https://api.ovh.com/console/#/dedicated/server/%7BserviceName%7D/biosSettings/sgx#GET` for example

Reply:
There's only one parameter which is `serviceName `.

Reply:
@antleblanc If this acts or will act like `/order/catalog/public/:product` maybe we can keep as is. This will allow to be more flexible as some options are added and we won't have to update this lib every time an option is added